### PR TITLE
Clean up the commutators of elements file

### PR DESCRIPTION
**Pull request by EgbertRijke**

### CODE REVIEW

**Comment by EgbertRijke** [comment]
I made some changes that require review:
- I changed the filename
- I made a new section `## External links` because the file contained an external link that I didn't just want to remove.

File: src/group-theory/commutators-of-elements-groups.lagda.md [hunk 1]
```diff
@@ -68,3 +66,7 @@ module _
   inv-commutator-Group x y =
     inv-right-div-Group G (mul-Group G x y) (mul-Group G y x)
 ```
+
+## External links
```
Comment (VojtechStep) [inline src/group-theory/commutators-of-elements-groups.lagda.md:70]: We have a `## References` header in other files, could you elaborate why this is different?

Comment (EgbertRijke) [reply src/group-theory/commutators-of-elements-groups.lagda.md:70]: References are for citations of articles. When you cite an article, then you do that with [n] in the text where you cite it, and then you record what [n] refers to under references. Then we refer to it by the title, author, journal, publisher, year, and all that stuff. Just like in an article. 

This is just an external link, and a citation in the above style seems inappropriate to me. As far as I could see, it is just a pointer to another online source on the same topic. To me, references carry more weight than just some external links. But of course this is up for debate, and maybe @fredrik-bakke also thinks it should just be called references.

Comment (fredrik-bakke) [reply src/group-theory/commutators-of-elements-groups.lagda.md:70]: I don't think our common patterns should restrict what we can call our sections, so I'm completely fine with `External links`

Comment (fredrik-bakke) [reply src/group-theory/commutators-of-elements-groups.lagda.md:70]: But references also don't have to be extremely formal. In #701, I use that section to link to formalizations I used as a starting point for instance

File: src/group-theory/commutators-of-elements-groups.lagda.md [hunk 2]
```diff
@@ -18,16 +18,14 @@ open import group-theory.groups
 
 ## Idea
 
-A commutator gives an indication of the extent to which a group multiplication
-fails to be commutative.
-
-The commutator of two elements, g and h, of a group G, is the element
+The **commutator** of two elements `g` and `h` of a
+[group](group-theory.groups.md) `G` is defined to be the element
 `[g, h] = (gh)(hg)⁻¹`.
```
Comment (fredrik-bakke) [inline src/group-theory/commutators-of-elements-groups.lagda.md:24]: I think the informal motivation could stay in a little rephrased form. Something like

```
The commutator can be seen as a correction factor that measures how much `g` and `h` fail to commute.
```

File: src/group-theory/commutators-of-elements-groups.lagda.md [hunk 3]
```diff
@@ -68,3 +66,7 @@ module _
   inv-commutator-Group x y =
     inv-right-div-Group G (mul-Group G x y) (mul-Group G y x)
 ```
+
+## External links
+
+- [Wikipedia](https://en.wikipedia.org/wiki/Commutator#Group_theory)
```
Comment (fredrik-bakke) [inline src/group-theory/commutators-of-elements-groups.lagda.md:72]: Should we include the title of the wikipedia article? It seems kind of nondescript in this form

**Review by fredrik-bakke — APPROVED** [review]

File: src/group-theory/commutators-of-elements-groups.lagda.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 # Commutators of elements in groups
```
Comment (fredrik-bakke) [inline src/group-theory/commutators-of-elements-groups.lagda.md:1]: Also, the file name is good. Don't want this to be confused with the commutator groups

File: src/group-theory/commutators-of-elements-groups.lagda.md (hunk 3 above)
Comment (EgbertRijke) [reply src/group-theory/commutators-of-elements-groups.lagda.md:72]: Yep, I will do that

File: src/group-theory/commutators-of-elements-groups.lagda.md (hunk 1 above)
Comment (EgbertRijke) [reply src/group-theory/commutators-of-elements-groups.lagda.md:70]: Note that Wikipedia also distinguishes between references, external links, and further reading.

**Comment by fredrik-bakke** [comment]
Looks good now. Merge when you're ready :)

**Pull request merged**